src/lib/capabilities.tsx

```diff
@@ -55,6 +55,9 @@ export type Capabilities = {
   editNotTracked: boolean
   accessEditor: boolean
   formatArticle: boolean
+  editorWrite: boolean
```

Comment:
It would be nice to continue having names of capabilities that mean an action (as in the rest of the items). Besides it's not clear what 'editorWrite' means.

Reply:
what do you think about write, we need this for https://github.com/Atypon-OpenSource/manuscripts-article-editor/blob/master/src/components/projects/lean-workflow/ManuscriptPageContainerLW.tsx#L145

I'm not good at naming, is there a name in your mind for this case

Reply:
ah, I see. Maybe just "editArticle" then? We don't need to keep 'write' in the naming. It's very old and was created before we had any permissions at all so it's very generic. 
Another thing - we isViewer there. But here you use isWriter. Is that correct?

Reply:
oh good catch updated it to `editArticle` with `!isViewer()`